com.thebrokenrail.MCPIReborn.yml

```diff
@@ -0,0 +1,193 @@
+app-id: com.thebrokenrail.MCPIReborn
+
+# MCPI Itself Is Proprietary
+tags:
+  - proprietary
+
+# Runtime/SDK
+runtime: org.freedesktop.Platform
+# Newer Runtimes Don't Support ARM Multiarch
+runtime-version: '19.08'
```

Comment:
given that this runtime is EOL, this won't fare well

Reply:
Not much I can really do about that, since newer runtimes don't have ARM multiarch or cross-compiler. And compiling `libc`, `libstdc++`, `ld-linux-armhf`, and an ARM cross-compiler sounds... not fun.